src/screens/index.js

```diff
@@ -1,31 +1,19 @@
 import AddParticipant from './AddParticipant';
+import AnonymousChannelListScreen from './AnonymousChannelListScreen';
 import ChannelListScreen from './ChannelListScreen';
-import ChannelListScreenV2 from './ChannelListScreenV2';
-import ChannelScreen from './ChannelScreen';
-import ChatDetailPage from './ChatDetailPage';
 import ContactScreen from './ContactScreen';
 import DetailDomainScreen from './DetailDomain/DetailDomainScreen';
-import DetailGroupImage from './DetailGroupImage';
 import FeedScreen from './FeedScreen';
-import GroupInfo from './GroupInfo';
-import GroupMedia from './GroupInfo/GroupMedia';
-import GroupSetting from './GroupSetting';
 import NewsScreen from './NewsScreen';
 import ProfileScreen from './ProfileScreen';
 
 export {
-  ChannelScreen,
   ChannelListScreen,
-  ChannelListScreenV2,
+  AnonymousChannelListScreen,
   FeedScreen,
   NewsScreen,
   ProfileScreen,
   ContactScreen,
   DetailDomainScreen,
-  ChatDetailPage,
-  DetailGroupImage,
-  GroupInfo,
-  GroupMedia,
-  AddParticipant,
-  GroupSetting
+  AddParticipant
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export of `AddParticipant` is present in both the new and old hunks, but it's not imported in the new hunk. This could lead to a runtime error if `AddParticipant` is not defined elsewhere in the file.
```diff
-   AddParticipant
+   // AddParticipant removed as it's not imported
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->